src/main/java/com/esri/samples/scene/get_elevation_at_a_point/GetElevationAtAPointSample.java

```diff
@@ -0,0 +1,179 @@
+                Point relativeSurfacePoint = sceneView.screenToBaseSurface(screenPoint);
+
+                // check that the point is on the surface, and primary button was clicked
+                if (relativeSurfacePoint != null && event.getButton() == MouseButton.PRIMARY && event.isStillSincePress()) {
+
+                    // clear any existing graphics from the graphics overlay
+                    graphicsOverlay.getGraphics().clear();
+
+                    // construct a polyline to use as a marker
+                    PolylineBuilder polylineBuilder = new PolylineBuilder(relativeSurfacePoint.getSpatialReference());
+                    Point baseOfPolyline = new Point(relativeSurfacePoint.getX(), relativeSurfacePoint.getY(), 0);
+                    polylineBuilder.addPoint(baseOfPolyline);
+                    Point topOfPolyline = new Point(baseOfPolyline.getX(), baseOfPolyline.getY(), 750);
+                    polylineBuilder.addPoint(topOfPolyline);
+                    Polyline markerPolyline = polylineBuilder.toGeometry();
+                    Graphic polylineGraphic = new Graphic(markerPolyline, lineSymbol);
+                    graphicsOverlay.getGraphics().add(polylineGraphic);
+
+                    // create a point to place the elevationTextSymbol above the polyline
+                    Point textSymbolPosition = new Point(topOfPolyline.getX(), topOfPolyline.getY(), topOfPolyline.getZ()+100);
+
+                    // get the surface elevation at the surface point
+                    ListenableFuture<Double> elevationFuture = scene.getBaseSurface().getElevationAsync(relativeSurfacePoint);
+                    elevationFuture.addDoneListener(() -> {
+                            try {
+                                // get the surface elevation
+                                Double elevation = elevationFuture.get();
+
+                                // update the text in the elevation marker
+                                updateMarkerText(elevationTextSymbol,textSymbolPosition,elevation);
+
+                            } catch (InterruptedException | ExecutionException e) {
+                                new Alert(Alert.AlertType.ERROR, e.getCause().getMessage()).show();
+                            }
+                        }
+                    );
+                }
+            });
+        } catch (Exception e) {
+            // on any error, display the stack trace.
+            e.printStackTrace();
+        }
+
+    }
+
+    /**
+     * Updates the elevationTextSymbol text
+     * @param elevationTextSymbol   The TextSymbol object to be updated
+     * @param position              The desired position of the TextSymbol Graphic
+     * @param elevation             The elevation to be inserted into the text;
+     */
+    private void updateMarkerText(TextSymbol elevationTextSymbol, Point position, double elevation){
+        elevationTextSymbol.setText((Math.round(elevation * 10d)/10d) + " m");
```

Comment:
I think rounding the elevation to the nearest meter will suffice since we're dealing with mountains here.